docs/src/documentation/05-development/01-guides/02-dictionary.mdx

```diff
@@ -1,3 +1,73 @@
 ---
 title: Dictionary
+redirect_from:
+  - /guides/dictionary/
 ---
+
+Our dictionary handles common translations like `Close` in components.
+
+Find our dictionary at: `@kiwicom/orbit-components/lib/data/dictionary/...`
+The files contain our own translations.
+
+## Example
+
+You can use dictionaries with various components.
+
+```jsx
+import en_GB from "@kiwicom/orbit-components/lib/data/dictionary/en-GB.json";
+import cs_CZ from "@kiwicom/orbit-components/lib/data/dictionary/cs-CZ.json";
+import Pagination from "@kiwicom/orbit-components/lib/Pagination";
+import ThemeProvider from "@kiwicom/orbit-components/lib/ThemeProvider";
+import defaultTheme from "@kiwicom/orbit-components/lib/defaultTheme";
+
```

Comment:
Looks like we are missing spacings between different parts of content, like between imports and types/component in examples here, can we fix it somehow? Looks like it trims all spaces 🤔

Reply:
A good point. Will standardize all examples once we have a clear way to present and use them in the docs.